src/controller/auth/logOut.js

```diff
@@ -0,0 +1,14 @@
+const logOut = (req, res) => {
+  const { token } = req.cookies;
+  if (token) {
+    res.clearCookie('token').status(200).json({
+      statusCode: 200,
+      massage: 'the user has been log out successfully',
+    });
+  } else {
+    res.status(401).json({
+      massage: 'authenticated',
```

Comment:
Suggest changing the 'authenticated' message to 'User is already logged out' or 'No active session found' when a user logs out without a token, for improved clarity and accuracy.